theme/templates/index.liquid

```diff
@@ -1,6 +1,7 @@
 {% assign show_featured_product = collections.frontpage.metafields.accentuate.show_featured_product %}
+{% assign news_feed_limit = shop.metafields.accentuate.news_feed_homepage_limit %}
 {% render 'home-header' %}
-{% render 'news-feed' %}
+{% render 'news-feed' is_limited: true, limit: news_feed_limit %}
```

Comment:
`shop.metafields.accentuate.news_feed_homepage_limit` = editor chooses a limit number for the homepage news feed